examples/preprocessing/plot_run_ica.py

```diff
@@ -25,23 +25,45 @@
 print(__doc__)
 
 ###############################################################################
-# Fit ICA model using the FastICA algorithm, detect and inspect components
+# Read and preprocess the data. Preprocessing consists of:
+# * meg channel selection
+# * 1 - 30 Hz band-pass IIR filter
+# * epoching -0.2 to 0.5 seconds with respect to events
 
 data_path = sample.data_path()
 raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
 
 raw = mne.io.read_raw_fif(raw_fname, preload=True)
-raw.filter(1, 30, method='iir')
 raw.pick_types(meg=True, eeg=False, exclude='bads', stim=True)
+raw.filter(1, 30, method='iir')
 
 # longer + more epochs for more artifact exposure
 events = mne.find_events(raw, stim_channel='STI 014')
 epochs = mne.Epochs(raw, events, event_id=None, tmin=-0.2, tmax=0.5)
 
+###############################################################################
+# Fit ICA model using the FastICA algorithm, detect and plot components
+# explaining ECG artifacts.
+
 ica = ICA(n_components=0.95, method='fastica').fit(epochs)
 
 ecg_epochs = create_ecg_epochs(raw, tmin=-.5, tmax=.5)
 ecg_inds, scores = ica.find_bads_ecg(ecg_epochs)
 
 ica.plot_components(ecg_inds)
+
+###############################################################################
+# Plot properties of ECG components:
 ica.plot_properties(epochs, picks=ecg_inds)
+
+###############################################################################
+# Mark EOG components for removal and compare evoked response before/after:
+
+ica.exclude = ecg_inds
+ica.plot_overlay(epochs.average())
+
+###############################################################################
+# Although the epochs evoked response does not change much, we can see the
+# effects of component removal clearly on `ecg_epochs`:
+
+ica.plot_overlay(ecg_epochs.average())
```

Comment:
Anyway - I changed the examples just to poke circle to render the images - so I can revert this change. However, IMO this is something you can't un-know, ICA doesn't do well in this example even if we resign from showing diagnostic plots... so I could do something about it anyway in subsequent PR (and revert this change for now). WDYT?